proto/worker_status.proto

```diff
@@ -0,0 +1,30 @@
+// Specification of the user facing gRPC API.
+syntax = "proto3";
+package status;
+
+service StatusApi {
+    // Returns the status of the worker.
+    rpc Status(StatusRequest) returns (StatusResponse) {}
+}
+
+// Request message for the status of the worker.
+message StatusRequest {}
+
+// Response message containing the status of the worker.
+message StatusResponse {
+    // Whether the worker is ready to generate proofs.
+    bool ready = 1;
```

Comment:
Does `ready` the worker is up or it is up and is not currently busy? At the very least, we should clarify this in the comments.

Reply:
`ready` means that the workers is up and receiving requests, it does not tells if the worker is busy or not.